uni/lib/model/app_state.dart

```diff
@@ -17,6 +17,7 @@ class AppState {
     return {
       'schedule': <Lecture>[],
       'exams': <Exam>[],
+      'hiddenExams': <Exam>[],
```

Comment:
```suggestion
      'hiddenExams': <String>[],
```

I ran the app with <Exam>, why it doesn't fail? 😟